packages/webapp/src/components/NotificationToast/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect } from 'react';
+import { useAppDispatch, useAppSelector } from '../../hooks';
+
+import toast, { Toaster } from 'react-hot-toast';
+import { BidTx, removePendingBidTx, removePendingSettleTx, resetPendingSettleTx, SettleTx } from '../../state/slices/mempool';
+import MempoolToast from '../MempoolToast';
+
+const NotificationToast: React.FC<{}> = props => {
+  const dispatch = useAppDispatch();
+
+  const closeToAuctionEnd = useAppSelector(state => state.auction.closeToEnd);
+  const pendingSettleTxs = useAppSelector(state => state.mempool.pendingSettleTxs);
+  const pendingBidTxs = useAppSelector(state => state.mempool.pendingBidTxs);
+  const prevSettledBlockHash = useAppSelector(state => state.settlement.prevSettledBlockHash);
+
+  const showToast = (tx: SettleTx | BidTx, time: number) => {
+    toast.custom(
+      <MempoolToast tx={tx} />,
+      {
+        position: "bottom-center",
+        duration: time
+      }
+    )
+  }
+
+  useEffect(() => {
+    pendingSettleTxs.forEach(tx => {
+      showToast(tx, 20000)
+      dispatch(removePendingSettleTx({ hash: tx.hash }));
+    })
+  }, [pendingSettleTxs, dispatch]);
+
+  useEffect(() => {
+    pendingBidTxs.forEach(tx => {
+      if (closeToAuctionEnd) {
```

Comment:
Do you think we still need this? Is there anything wrong with showing incoming bids when the auction is not close to ending?

Reply:
Maybe it's useful to show in the `<MempoolToast>`... See above comment!

Reply:
Yes, I think. As bids that come when `time > 5min` doesn't change anything from the Fomo standpoint, there is no real reason to show it. I think if we were showing the current bids for the noun on the site, then it would be helpful.